src/features/courses/course-route.ts

```diff
@@ -603,6 +603,57 @@ router.post('/question',
         }
     }));
 
+router.get('/question/:id/raw',
```

Comment:
What is meant by get a raw question? I think we need a method comment on this one

Reply:
we had branching logic in get question - this is just getting the question fields from the db table.